server/container/guice/protocols/jmap/pom.xml

```diff
@@ -41,6 +41,10 @@
             <artifactId>apache-mailet-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-store</artifactId>
```

Comment:
why ? It's not what the commit says

Reply:
The binding for the systemMailboxprovider impl is node here, I will move it to a more generic module